composer.json

```diff
@@ -75,6 +75,7 @@
         "drupal/metatag": "1.16",
         "drupal/migrate_plus": "5.1",
         "drupal/migrate_tools": "5.0",
+        "drupal/migrate_queue_importer": "2.0.0",
```

Comment:
Is this more specific of a version number than we want to pin to?

Reply:
Forgot about this comment.